### DISCUSSION TITLE
Unable to See Sensors or Set Alarm

### DISCUSSION THREAD

**Discussion by vipul128** [tsightler/ring-mqtt#556] [Support Q&A] [answered]
I just installed the MQTT-Ring add-on and am having issues seeing my sensors and being able to set my alarm.  I must be missing something, but I'm not sure exactly what since I followed the instructions.  Right now I only see "Info" and "Tamper" as entities under my Ring Alarm device.  Is there any way to see all the other sensors and keypad and most importantly arm and disarm?  I know for sure I'm missing something, so some guidance would be appreciated.  Thanks!

**Comment by tsightler** [comment, marked as answer]
Could you provide a screenshot or something to go on.  Almost seems like you are viewing the base station and not the alarm control panel.

**Reply by tsightler** [reply]
Also, logs would be really useful, especially during startup when all discovered devices are listed.

**Reply by vipul128** [reply]
The logs showed all devices being detected.  I guess I just wasn't patient enough.  I rebooted the HA instance after about 30 mins and the devices and entities all showed up properly.

**Reply by tsightler** [reply]
It shouldn't take more than a minute or two for devices to appear, but I've seen some weird behavior with the HA UI when there's lots of discovered devices in rapid succession.  Usually forcing a hard browser refresh is enough to get the devices to appear in that case.
